app/controllers/merchants_controller.rb

```diff
@@ -0,0 +1,48 @@
+class MerchantsController < ApplicationController
+  def index
+    @merchants = Merchant.all
+  end
+
+  def show
+    @merchant = Merchant.find(params[:id])
+  end
+
+  def new
+    # @merchant = Merchant.new
```

Comment:
delete this commented code.. you're right, we don't need a merchant object since that's what we're trying to create in the new form.